src/verify_config.py

```diff
@@ -42,6 +42,7 @@ def print_help():
     PARAM.append("CHIMES_LSQ");                     VARTYP.append("str");           DETAILS.append("ChIMES_lsq executable absolute path (e.g. CHIMES_SRCDIR + \"chimes_lsq\")")
     PARAM.append("CHIMES_SOLVER");                  VARTYP.append("str");           DETAILS.append("lsq2.py executable absolute path (e.g. CHIMES_SRCDIR + \"lsq2.py\")")
     PARAM.append("CHIMES_POSTPRC");                 VARTYP.append("str");           DETAILS.append("post_proc_lsq2.py executable absolute path (e.g. CHIMES_SRCDIR + \"post_proc_lsq2.py\")")
+    PARAM.append("CHIMES_MODULES");                 VARTYP.append("str");           DETAILS.append("chimes modules directory")
```

Comment:
This is not the correct fix for this problem. The problem is a typo in main.py (see my comment there). Hence, revert this change.

Reply:
Delete this line (45) from your PR

Reply:
Delete this line (45) from your PR